x-pack/test_serverless/api_integration/test_suites/common/platform_security/api_keys.ts

```diff
@@ -100,9 +112,11 @@ export default function ({ getService }: FtrProviderContext) {
           });
           expect(status).toBe(400);
 
-          ({ body, status } = await supertest
+          ({ body, status } = await supertestWithoutAuth
             .put('/internal/security/api_key')
             .set(svlCommonApi.getInternalRequestHeader())
+            // .set(roleAuthc.apiKeyHeader)
```

Comment:
debug leftover?

Reply:
Oh dang, on it.

Reply:
https://github.com/elastic/kibana/pull/184935/commits/efd30b40790aae5346c2fb43d190369b2dba8c80